src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,10 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+2.0.0b8
```

Comment:
This should be a breaking change. According to [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md), please choose `3.0.0b1`.